src/ingestors/fxhash/abi.ts

```diff
@@ -1,125 +1,4 @@
 export const FXHASH_BASE_FRAME_ABI = [
-  {
```

Comment:
For overall sanity and neatness I wonder if we should think about a convention for ABI's? 

Move them to an external `abi.ts` or `abi.json` in the ingestor directory?